src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordCopyCtor.cs

```diff
@@ -55,6 +57,15 @@ internal override void GenerateMethodBodyStatements(SyntheticBoundNodeFactory F,
             }
         }
 
+        internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes)
+        {
+            base.AddSynthesizedAttributes(moduleBuilder, ref attributes);
+            Debug.Assert(IsImplicitlyDeclared);
+            var compilation = this.DeclaringCompilation;
+            AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_Runtime_CompilerServices_CompilerGeneratedAttribute__ctor));
+            Debug.Assert(WellKnownMembers.IsSynthesizedAttributeOptional(WellKnownMember.System_Runtime_CompilerServices_CompilerGeneratedAttribute__ctor));
+        }
```

Comment:
I can see a room for refactoring, since this block is repeated in many places (not only in this PR - but over the codebase)

For example, we can have a virtual property in `Symbol`:

```csharp
protected virtual bool SynthesizeCompilerGeneratedAttribute => false;
```

and in the already-existing empty virtual `AddSynthesizedAttributes` method:

```csharp
if (SynthesizeCompilerGeneratedAttribute)
{
    // Add this repeated code block here.
}
```

Let me know if the refactoring worth it, and (if yes) whether you want it in this PR or a separate one.